.github/workflows/build.yml

```diff
@@ -361,6 +361,48 @@ jobs:
           name: macos-dmg
           path: |
             app/linwood-butterfly-macos.dmg
+
+  build-ipa:
+    name: Build IPA (iOS)
+    runs-on: macos-latest
+    defaults:
+      run:
+        working-directory: app
+    steps:
+      - name: ⬆️ Checkout
+        uses: actions/checkout@v4
+      - name: Get flutter version
+        run: |
+          FLUTTER_VERSION=$(cat ../FLUTTER_VERSION)
+          echo "FLUTTER_VERSION=${FLUTTER_VERSION}" >> $GITHUB_ENV
+      - uses: subosito/flutter-action@v2
+        with:
+          flutter-version: ${{ env.FLUTTER_VERSION }}
+          channel: 'master'
+      - name: 📦 Get dependencies
+        run: |
+          flutter clean
+          flutter pub get
+          flutter doctor -v
+      - name: 🏭 Build
+        run: |
+          flutter build ios --release --no-codesign
+      - name: Packing IPA
+        run: |
+          cd build/ios/iphoneos/
+          rm -rf Payload
+          mkdir Payload
+          cp -R Runner.app Payload/
+          rm -f linwood-butterfly-iOS.ipa
+          zip -vr linwood-butterfly-iOS.ipa Payload/
+          ls -l linwood-butterfly-iOS.ipa
```

Comment:
it needs also be changed here, i would change it manually

Reply:
✅